lib/stripe/profile_storer.rb

```diff
@@ -65,7 +65,7 @@ def source_attrs_from(response)
     def customer_profile_id(response)
       response.params['customer'] || response.params['id']
     end
-
+    
```

Comment:
you can revert the changes in this file so the commit history is clean.
are these new spaces correct?